packages/constants/src/network/network.constant.ts

```diff
@@ -12,26 +12,44 @@ export enum NetworkServer {
   AMM_DEVNET = 'wss://amm.devnet.rippletest.net:51233'
 }
 
-export const NETWORK = {
-  [Network.MAINNET]: {
-    name: Network.MAINNET,
-    server: NetworkServer.MAINNET,
-    description: 'Main network using the production version of the XRP Ledger.'
-  },
-  [Network.TESTNET]: {
-    name: Network.TESTNET,
-    server: NetworkServer.TESTNET,
-    description:
-      'Acts as a testing network, without impacting production users and risking real money.'
-  },
-  [Network.DEVNET]: {
-    name: Network.DEVNET,
-    server: NetworkServer.DEVNET,
-    description: 'A preview of upcoming features, where unstable changes are tested out.'
-  },
-  [Network.AMM_DEVNET]: {
-    name: Network.AMM_DEVNET,
-    server: NetworkServer.AMM_DEVNET,
-    description: 'XLS-30d Automated Market Makers preview network.'
-  }
-};
+export interface NetworkInfo {
+  name: Network;
```

Comment:
While refactoring the NetworkContext, I think we should also refactor the `Network` enum and replace it with strings; eg. `type Network = 'Mainnet' | 'Testnet'...`

Reply:
why ?